overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -34346,7 +34346,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "Being able to fly is pretty sick. Its also pretty expensive.\n\nIts probably a bit premature to consider a §6jetpack§r at this point, but if you really want to craft this bad boy, I won\u0027t stop you.\n\nThere\u0027s also a quest chain for §bEnderIO§r Jetpacks (starting with the §6Conductive Iron Jetpack§r). Eventually you\u0027ll need to make both, but not for a long time, so feel free to ignore at least one of these quest chains unless you really really like flying.",
+          "desc:8": "Being able to fly is pretty sick. It\u0027s also pretty expensive.\n\nIt\u0027s probably a bit premature to consider a §6jetpack§r at this point, but if you really want to craft this bad boy, I won\u0027t stop you.\n\nThere\u0027s also a quest chain for §bEnderIO§r Jetpacks (starting with the §6Conductive Iron Jetpack§r). Eventually you\u0027ll need to make both, but not for a long time, so feel free to ignore at least one of these quest chains unless you really like flying.",
```

Comment:
The corrections of the contraction are correct, but the duplicated "really" is intentional.

Reply:
Reverted "really" change